Lib/ufo2ft/outlineOTF.py

```diff
@@ -380,6 +406,21 @@ def setupTable_cmap(self):
             cmap12_3_10.cmap = nonBMP
             # update tables registry
             cmap.tables = [cmap4_0_3, cmap4_3_1, cmap12_0_4, cmap12_3_10]
+        if self.macLegacy:
+            from fontTools.ttLib.tables._c_m_a_p import cmap_format_6
```

Comment:
Does anyone know why we have these cmap imports inline? I'd prefer them at the top of the file if possible.